device_operation/include/device_gemm_xdl_c_shuffle.hpp

```diff
@@ -43,13 +43,15 @@ template <
     ck::index_t ABlockTransferSrcVectorDim,
     ck::index_t ABlockTransferSrcScalarPerVector,
     ck::index_t ABlockTransferDstScalarPerVector_K1,
+    bool AThreadTransferSrcResetCoordinateAfterRun,
```

Comment:
Do we need to expose `A/BThreadTransferSrcResetCoordinateAfterRun` parameters to the user? Because the "C_Shuffle" version is supposed to be similar to the vanilla one except for the C buffer part, and I don't see `A/BThreadTransferSrcResetCoordinateAfterRun`  exposed in the vanilla one that you just replaced in #71 

Would you also take a look at my question posted here and address it? https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/71#discussion_r789494895

Reply:
I think you are rgiht. Already fix it.

Reply:
Repost: https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/7e262f62002cf5277d8895c87ef3ac4d8b581330/device_operation/include/device_gemm_xdl_c_shuffle.hpp#L320
> Is `v2r3` a typo here, since you were invoking `v3r1` kernel when condition is true? @asroy can you confirm if this is intended?